File: sem7/src/main/java/ru/gb/oseminar/data/Student.java
package ru.gb.oseminar.data;

public class Student {

    public String firstName;
    public String lastName;
    public String phone;
    public String dateOfBirth;
    public StudentType type;

    public Student(String firstName, String lastName, String phone, String dateOfBirth, StudentType type) {
        this.firstName = firstName;
        this.lastName = lastName;
        this.phone = phone;
        this.dateOfBirth = dateOfBirth;
        this.type = type;
    }

    public String getFirstName() {
        return firstName;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    public String getPhone() {
        return phone;
    }

    public void setPhone(String phone) {
        this.phone = phone;
    }

    public String getDateOfBirth() {
        return dateOfBirth;
    }

    public void setDateOfBirth(String dateOfBirth) {
        this.dateOfBirth = dateOfBirth;
    }

    public StudentType getType() {
        return type;
    }

    public void setType(StudentType type) {
        this.type = type;
    }

    @Override
    public String toString() {
        return "Student [firstName=" + firstName + ", lastName=" + lastName + ", phone=" + phone + ", dateOfBirth="
                + dateOfBirth + ", type=" + type + "]";
    }
}

File: FinalHW-OOP/data/Student.cs
namespace FinalHWOOP {

    public class Student {
        public string firstName;
        public string secondName;
        public string phone;
        public string dateOfBirth;
        public Type type;

        public Student(string firstName, string secondName, string phone, string dateOfBirth, Type type) {
            this.firstName = firstName;
            this.secondName = secondName;
            this.phone = phone;
            this.dateOfBirth = dateOfBirth;
            this.type = type;
        }

        public void setFirstName(String firstName) {
            this.firstName = firstName;
        }

        public String getFirstName() {
            return this.firstName;
        }

        public void setSecondName(String secondName) {
            this.secondName = secondName;
        }

        public String getSecondName() {
            return this.secondName;
        }

        public void setPhone(String phone) {
            this.phone = phone;
        }

        public String getPhone() {
            return this.phone;
        }

        public void setDateOfBirth(String dateOfBirth) {
            this.dateOfBirth = dateOfBirth;
        }

        public String getdateOfBirth() {
            return this.dateOfBirth;
        }

        public void setType(Type type) {
            this.type = type;
        }

        public Type getType() {
            return this.type;
        }

        public override string ToString() {
            return "Student - " + this.firstName + " " + this.secondName + ", phone: " + this.phone
                + ", birth: " + this.dateOfBirth + ", type=[" + this.type + "];";
        }
    }
}
